pkg/apis/scope/v0alpha1/types.go

```diff
@@ -73,3 +73,30 @@ type ScopeDashboardBindingList struct {
 
 	Items []ScopeDashboardBinding `json:"items,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type ScopeNode struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ScopeNodeSpec `json:"spec,omitempty"`
+}
+
+type ScopeNodeSpec struct {
+	//+optional
+	ParentName   *string `json:"parentName"`
```

Comment:
Does this need to be a pointer? Would empty string struffice for "No parent", or does null and empty have different meanings?